easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -1105,15 +1105,19 @@ def filter_deps(self, deps):
 
         return retained_deps
 
-    def dependencies(self, build_only=False):
+    def dependencies(self, build_only=False, runtime_only=False):
```

Comment:
Can't we have either an "enum" `dep_type=BUILD|RUNTIME|BOTH` or at least use `include_build=True, include_runtime=True` because now we have a potentially "invalid" call: When both are True

Maybe we should at least rename to `exclude_runtime, exclude_build` which would be mostly backwards compatible and setting both to True seems more reasonable to return nothing

Reply:
Let's follow up on that in a separate PR, I want to get this PR merged ASAP.

Maybe open an issue for it, so we can tag it with `5.0` milestone.